mne/fixes.py

```diff
@@ -1250,6 +1252,25 @@ def bincount(x, weights, minlength):  # noqa: D103
             out[idx] += w
         return out
 
+    # fix because Numba does not support axis kwarg for mean
+    @jit()
+    def _np_apply_along_axis(func1d, axis, arr):
+        assert arr.ndim == 2
+        assert axis in [0, 1]
+        if axis == 0:
+            result = np.empty(arr.shape[1])
+            for i in range(len(result)):
+                result[i] = func1d(arr[:, i])
+        else:
+            result = np.empty(arr.shape[0])
+            for i in range(len(result)):
+                result[i] = func1d(arr[i, :])
+        return result
+
+    @jit()
+    def mean(array, axis):
+        return _np_apply_along_axis(np.mean, axis, array)
```

Comment:
is it really faster then doing np.mean with axis directly?

Reply:
No it's the same speed. But this is a necessary shim to be able to do `np.mean(..., axis=...)` *inside* Numba `jit`-ed functions. And `jit`ting those functions (which require `np.mean(..., axis=...)` support) is substantially faster than staying in pure Python.

Someday when Numba adds support for the `axis` argument, we can remove this shim